Makefile

```diff
@@ -141,7 +141,7 @@ test-npm: $(NODE_EXE)
 	NODE_EXE=$(NODE_EXE) tools/test-npm.sh
```

Comment:
Yes, I unrelatedly figured out that this was broken after quickly discussing it at nodeconf. I'll open a PR.